src/test/unit/process-profile.test.js

```diff
@@ -664,8 +664,14 @@ describe('profile meta processing', function () {
 });
 
 describe('visualMetrics processing', function () {
-  function checkVisualMetricsForThread(thread: Thread) {
-    const metrics = [
+  function checkVisualMetricsForThread(
+    thread: Thread,
+    expectedMetricMarkers?: Array<{|
+      name: string,
+      changeMarkerLength: number,
+    |}>
```

Comment:
optional: I would suggest to make this property mandatory and pass them directly in the existing tests.  The array looks small enough that it can be repeated.